packages/react-composites/src/composites/CallComposite/pages/ConfigurationPage.tsx

```diff
@@ -157,18 +163,8 @@ export const ConfigurationPage = (props: ConfigurationPageProps): JSX.Element =>
   };
   /* @conditional-compile-remove(call-readiness) */
   const networkErrors = errorBarProps.activeErrorMessages.filter((message) => message.type === 'callNetworkQualityLow');
-
-  /* @conditional-compile-remove(call-readiness) */
-  const [isDrawerShowing, setIsDrawerShowing] = useState(true);
-  /* @conditional-compile-remove(call-readiness) */
-  const onLightDismissTriggered = (): void => {
-    // do nothing here
-    // only way to dismiss this drawer is clicking on allow access which will leads to device permission prompt
-  };
-
   /* @conditional-compile-remove(call-readiness) */
-  const [isModalShowing, setIsModalShowing] = useState(false);
-
+  const [isModalShowing, setIsModalShowing] = useState(true);
```

Comment:
Is this always showing by default? what if they have accepted permissions previously, or if they have opted out of call readiness?

Reply:
we use the video and audio state to determine which modal to show or show nothing, this is just for making modal disappear when dismissed

Reply:
Can you update the name to reflect this? If someone is reading this or debugging they will see this as true and expect a modal to be showing.
Something like hasPermissionsModalBeenDismissed would better reflect this variables purpose

Reply:
made changes and pushed to PR https://github.com/Azure/communication-ui-library/pull/2529